### ISSUE TITLE
netdata.spec seems to reference missing files

### ISSUE THREAD

**Issue by ElCoyote27** [netdata/netdata#4409]
Hi,
Thanks for #4400 . I'm hitting another issue at the end of the RPM build on el7.5:
+ /usr/lib/rpm/find-debuginfo.sh --strict-build-id -m --run-dwz --dwz-low-mem-die-limit 10000000 --dwz-max-die-limit 110000000 /export/home/raistlin/World/Vincent/Code/redhat/BUILD/netdata-1.10.0-779-g4f84553_rolling
extracting debug info from /export/home/raistlin/World/Vincent/Code/redhat/BUILDROOT/netdata-1.10.1-20181016.el7.x86_64/usr/libexec/netdata/plugins.d/cgroup-network
extracting debug info from /export/home/raistlin/World/Vincent/Code/redhat/BUILDROOT/netdata-1.10.1-20181016.el7.x86_64/usr/libexec/netdata/plugins.d/freeipmi.plugin
extracting debug info from /export/home/raistlin/World/Vincent/Code/redhat/BUILDROOT/netdata-1.10.1-20181016.el7.x86_64/usr/libexec/netdata/plugins.d/apps.plugin
extracting debug info from /export/home/raistlin/World/Vincent/Code/redhat/BUILDROOT/netdata-1.10.1-20181016.el7.x86_64/usr/sbin/netdata
/usr/lib/rpm/sepdebugcrcfix: Updated 4 CRC32s, 0 CRC32s did match.
4554 blocks
+ /usr/lib/rpm/check-buildroot
Processing files: netdata-1.10.1-20181016.el7.x86_64
error: File not found by glob: /export/home/raistlin/World/Vincent/Code/redhat/BUILDROOT/netdata-1.10.1-20181016.el7.x86_64/usr/lib64/netdata/*.conf
error: File not found by glob: /export/home/raistlin/World/Vincent/Code/redhat/BUILDROOT/netdata-1.10.1-20181016.el7.x86_64/usr/lib64/netdata/charts.d/*.conf
error: File not found by glob: /export/home/raistlin/World/Vincent/Code/redhat/BUILDROOT/netdata-1.10.1-20181016.el7.x86_64/usr/lib64/netdata/health.d/*.conf
error: File not found by glob: /export/home/raistlin/World/Vincent/Code/redhat/BUILDROOT/netdata-1.10.1-20181016.el7.x86_64/usr/lib64/netdata/node.d/*.conf
error: File not found by glob: /export/home/raistlin/World/Vincent/Code/redhat/BUILDROOT/netdata-1.10.1-20181016.el7.x86_64/usr/lib64/netdata/python.d/*.conf
error: File not found by glob: /export/home/raistlin/World/Vincent/Code/redhat/BUILDROOT/netdata-1.10.1-20181016.el7.x86_64/usr/lib64/netdata/statsd.d/*.conf
error: File not found by glob: /export/home/raistlin/World/Vincent/Code/redhat/BUILDROOT/netdata-1.10.1-20181016.el7.x86_64/usr/lib64/netdata/node.d/*.md
error: Directory not found: /export/home/raistlin/World/Vincent/Code/redhat/BUILDROOT/netdata-1.10.1-20181016.el7.x86_64/etc/netdata/health.d
error: Directory not found: /export/home/raistlin/World/Vincent/Code/redhat/BUILDROOT/netdata-1.10.1-20181016.el7.x86_64/usr/lib64/netdata/health.d
error: Directory not found: /export/home/raistlin/World/Vincent/Code/redhat/BUILDROOT/netdata-1.10.1-20181016.el7.x86_64/etc/netdata/python.d
error: Directory not found: /export/home/raistlin/World/Vincent/Code/redhat/BUILDROOT/netdata-1.10.1-20181016.el7.x86_64/usr/lib64/netdata/python.d
error: Directory not found: /export/home/raistlin/World/Vincent/Code/redhat/BUILDROOT/netdata-1.10.1-20181016.el7.x86_64/etc/netdata/charts.d
error: Directory not found: /export/home/raistlin/World/Vincent/Code/redhat/BUILDROOT/netdata-1.10.1-20181016.el7.x86_64/usr/lib64/netdata/charts.d
error: Directory not found: /export/home/raistlin/World/Vincent/Code/redhat/BUILDROOT/netdata-1.10.1-20181016.el7.x86_64/etc/netdata/node.d
error: Directory not found: /export/home/raistlin/World/Vincent/Code/redhat/BUILDROOT/netdata-1.10.1-20181016.el7.x86_64/usr/lib64/netdata/node.d
error: Directory not found: /export/home/raistlin/World/Vincent/Code/redhat/BUILDROOT/netdata-1.10.1-20181016.el7.x86_64/etc/netdata/statsd.d
error: Directory not found: /export/home/raistlin/World/Vincent/Code/redhat/BUILDROOT/netdata-1.10.1-20181016.el7.x86_64/usr/lib64/netdata/statsd.d
Executing(%doc): /bin/sh -e /var/tmp/rpm-tmp.fGI7Mw
+ umask 022
+ cd /export/home/raistlin/World/Vincent/Code/redhat/BUILD
+ cd netdata-1.10.0-779-g4f84553_rolling
+ DOCDIR=/export/home/raistlin/World/Vincent/Code/redhat/BUILDROOT/netdata-1.10.1-20181016.el7.x86_64/usr/share/doc/netdata-1.10.1
+ export DOCDIR
+ /usr/bin/mkdir -p /export/home/raistlin/World/Vincent/Code/redhat/BUILDROOT/netdata-1.10.1-20181016.el7.x86_64/usr/share/doc/netdata-1.10.1
+ cp -pr README.md /export/home/raistlin/World/Vincent/Code/redhat/BUILDROOT/netdata-1.10.1-20181016.el7.x86_64/usr/share/doc/netdata-1.10.1
+ exit 0

Not sure if I can fix that myself as I don't know the netdata code base well enough..

**Comment by ktsaou** [comment]
I followed this procedure: https://github.com/netdata/netdata/pull/1819#issuecomment-282583932

And got this output:

```
RPM build errors:
    File not found by glob: /root/rpmbuild/BUILDROOT/netdata-1.10.1-1.el6.x86_64/usr/lib64/netdata/*.conf
    File not found by glob: /root/rpmbuild/BUILDROOT/netdata-1.10.1-1.el6.x86_64/usr/lib64/netdata/charts.d/*.conf
    File not found by glob: /root/rpmbuild/BUILDROOT/netdata-1.10.1-1.el6.x86_64/usr/lib64/netdata/health.d/*.conf
    File not found by glob: /root/rpmbuild/BUILDROOT/netdata-1.10.1-1.el6.x86_64/usr/lib64/netdata/node.d/*.conf
    File not found by glob: /root/rpmbuild/BUILDROOT/netdata-1.10.1-1.el6.x86_64/usr/lib64/netdata/python.d/*.conf
    File not found by glob: /root/rpmbuild/BUILDROOT/netdata-1.10.1-1.el6.x86_64/usr/lib64/netdata/statsd.d/*.conf
    File not found by glob: /root/rpmbuild/BUILDROOT/netdata-1.10.1-1.el6.x86_64/usr/lib64/netdata/node.d/*.md
    File not found: /root/rpmbuild/BUILDROOT/netdata-1.10.1-1.el6.x86_64/etc/netdata/health.d
    File not found: /root/rpmbuild/BUILDROOT/netdata-1.10.1-1.el6.x86_64/usr/lib64/netdata/health.d
    File not found: /root/rpmbuild/BUILDROOT/netdata-1.10.1-1.el6.x86_64/etc/netdata/python.d
    File not found: /root/rpmbuild/BUILDROOT/netdata-1.10.1-1.el6.x86_64/usr/lib64/netdata/python.d
    File not found: /root/rpmbuild/BUILDROOT/netdata-1.10.1-1.el6.x86_64/etc/netdata/charts.d
    File not found: /root/rpmbuild/BUILDROOT/netdata-1.10.1-1.el6.x86_64/usr/lib64/netdata/charts.d
    File not found: /root/rpmbuild/BUILDROOT/netdata-1.10.1-1.el6.x86_64/etc/netdata/node.d
    File not found: /root/rpmbuild/BUILDROOT/netdata-1.10.1-1.el6.x86_64/usr/lib64/netdata/node.d
    File not found: /root/rpmbuild/BUILDROOT/netdata-1.10.1-1.el6.x86_64/etc/netdata/statsd.d
    File not found: /root/rpmbuild/BUILDROOT/netdata-1.10.1-1.el6.x86_64/usr/lib64/netdata/statsd.d
```

I am trying to fix it in PR #4410

**Comment by ktsaou** [comment]
Please retry. It has been fixed.
